terraform/modules/common/ami/main.tf

```diff
@@ -1,8 +1,12 @@
 ## Variables
 
+locals {
+  canonical_account_id = "099720109477"
+}
+
 ## Data sources
 
-data "aws_ami" "source" {
+data "aws_ami" "ecs_optimized" {
```

Comment:
Why has this changed to be `ecs_optimized` when the rest of the PR is for EC2 Prometheis?

Reply:
see commit message for 6ec9817c2debe168ac6cb96753fcf4ba38534e0d

Reply:
Oh, I didn't look at the commits individually.